akka-actor/src/main/scala/akka/routing/Resizer.scala

```diff
@@ -273,11 +295,13 @@ private[akka] final class ResizablePoolCell(
       resizer.isTimeForResize(resizeCounter.getAndIncrement()) && resizeInProgress.compareAndSet(false, true)) {
       super.sendMessage(Envelope(ResizablePoolActor.Resize, self, system))
     }
+
     super.sendMessage(envelope)
   }
 
   private[akka] def resize(initial: Boolean): Unit = {
     if (resizeInProgress.get || initial) try {
+      tryReportMessageCount()
```

Comment:
is this the only way? can't it be done inside the ordinary `resizer.resize`?  you need the counter? can it have it's own counter?